app/src/main/java/com/chess/clock/activities/SettingsActivity.java

```diff
@@ -308,4 +321,18 @@ public void saveTimeControl() {
     public void onTimeControlListEmpty() {
         Toast.makeText(this, getString(R.string.list_empty_toast_message), Toast.LENGTH_LONG).show();
     }
+
+    public void setNavigationOnItemSelectedListener(OnNavigationItemSelectedListener listener) {
+        if (mBottomNavigationView != null) {
+            mBottomNavigationView.setOnNavigationItemSelectedListener(listener);
+        }
+    }
+
+    public void setBottomNavigationViewVisibility(int v) {
+        mBottomNavigationView.setVisibility(v);
```

Comment:
With a null check in `setNavigationOnItemSelectedListener` I feel it would also be appropriate to have a null check here in this method.